types/braintree/index.d.ts

```diff
@@ -647,10 +650,10 @@ declare namespace braintree {
     export interface MerchantAccountCreateRequest {
         business?: MerchantBusiness;
         funding: MerchantFunding;
-        id: string;
+        id?: string;
         individual: MerchantIndividual;
         masterMerchantAccountId: string;
-        status: MerchantAccountStatus;
+        status?: MerchantAccountStatus;
```

Comment:
Just did another check of the docs, I am seeing it on the response interface, but not the request.

Reply:
The `status` field only exists on the response interface, so rather than remove it I had changed it to optional but I'll go ahead and remove it from the request interface.